### ISSUE TITLE
Dynamic daytimes

### ISSUE THREAD

**Issue by freekeys** [benleb/ad-automoli#40]
I'd love to have the daytime adjustments be dynamic based on sunset time (e.g. sunset-1hr, sunset-30mins etc etc)

**Comment by oscfor** [comment]
Yeah, that would be great. Up in the north we have great difference in sunrise during the year.

**Comment by sivastola** [comment]
I'd love to see something like this as well.
Here is how I used it in automations.yaml before I moved to automoli
```
id: aqara_motion2_an
  alias: BWM Aqara Motion2 Flurlicht An
  trigger:
  - entity_id: group.motiongroupflurlicht
    platform: state
    to: 'on'
  action:
  - entity_id: light.flurlicht
    service: homeassistant.turn_on
  condition:
    condition: or
    conditions:
    - condition: sun
      after: sunset
      after_offset: -01:00:00
    - condition: sun
      before: sunrise
      before_offset: 01:00:00
```
the important part is the condition-part
maybe it helps

**Comment by paviro** [comment]
@freekeys @oscfor @sivastola if you want you can try my commit and report back if you run into any issues.

**Comment by oscfor** [comment]
> @freekeys @oscfor @sivastola if you want you can try my commit and report back if you run into any issues.

Just switched both a wave and zigbee platform, so it will take a few days until I am stable enough to do real testing.

**Comment by benleb** [comment]
will check and probably merge this in the next 1-2 days

**Comment by paviro** [comment]
Any updates on this?

**Comment by benleb** [comment]
implemented by @paviro in #60 - thank you! 🎉

**Comment by paviro** [comment]
Thanks for merging! ☺️
